bliss/encoder/plotting.py

```diff
@@ -112,3 +112,79 @@ def plot_detections(images, true_tile_cat, est_tile_cat, margin_px, ticks=None,
 
     fig.tight_layout()
     return fig
+
+
+# Adapted from plot_detections in bliss.plotting
+def my_plot_detections(
```

Comment:
Did changes to this file get include accidentally?

Reply:
Oh, this file is also modifed by undergraduates. I will delete all the changes made by them.

Reply:
Sounds good. Let's add their code in our next PR, which will address issue #976